sql/core/src/test/resources/sql-tests/results/postgreSQL/with.sql.out

```diff
@@ -223,32 +223,30 @@ Table or view not found: outermost; line 4 pos 23
 
 
 -- !query
-CREATE TABLE withz USING parquet AS SELECT i AS k, CAST(i AS string) || ' v' AS v FROM (SELECT EXPLODE(SEQUENCE(1, 16, 3)) i)
+CREATE TABLE withz USING parquet AS SELECT i AS k, CAST(i || ' v' AS string) v FROM (SELECT EXPLODE(SEQUENCE(1, 16, 3)) i)
```

Comment:
I see what's going on now. This test is for the SQL WITH feature, not string concat, and we lose test coverage because we failed to create the table and following queries all fail.

Let's change it to `CAST(i AS string) || ' v'`.

Reply:
yeah, that was my purpose.